packages/shared/src/components/modals/common/ModalClose.tsx

```diff
@@ -19,11 +17,9 @@ function ModalCloseComponent(
   return (
     <CloseButton
       {...props}
-      buttonSize={buttonSize}
       onClick={onClick}
       ref={ref}
-      className={classNames('right-2 z-1', className)}
-      style={{ position: 'absolute', ...style }}
+      className={classNames('right-2 z-1', { absolute }, className)}
```

Comment:
Perhaps we can have the position as a default prop?
`right-2` and then override for others?

Reply:
I just did that in [1b2d07a](https://github.com/dailydotdev/apps/pull/2435/commits/1b2d07a83d32f72e5b63a607d860a921955545b3)